packages/babel-core/src/transformation/internal-plugins/block-hoist.js

```diff
@@ -1,7 +1,6 @@
-import Plugin from "../plugin";
 import sortBy from "lodash/sortBy";
 
-export default new Plugin({
```

Comment:
Oh interesting, didn't even know you could return an object, I see it's at https://github.com/loganfsmyth/babel/blob/bf13ed4da0282aa3fce8f2805c23df35f26cfd80/packages/babel-core/src/transformation/file/options/option-manager.js#L105